### ISSUE TITLE
Possible to move/adjust UI elements?

### ISSUE THREAD

**Issue by DeaconMidgard** [Lyall/RF5Fix#4]
This works great on 32:9 but it does make the UI elements stick out a bit more. The minimap is the worst offender. Would it be possible to realign these some way?

**Comment by Lyall** [comment]
Yes it's possible but unless there is a better method then it would involve moving individual UI elements one by one then it would be a rather difficult process. You'd have to shift each part of the UI by specific amounts. So for that reason it's probably not something that would I would take on doing. I will look into it a bit further and see if there's a better way but it may go nowhere.

 TL;DR it's possible but unlikely that I would do it for RF5Fix.

**Comment by DeaconMidgard** [comment]
Makes sense. It looks fine as is for the most part. I only cringe when I expand the minimap which is only for quick checks so it is definitely not something needed if the effort is a lot of work when it's a feature that is really not needed. I appreciate the update and thanks for doing RF5Fix. It fixed some stuttering issues I was having so definitely enjoying it!

**Comment by FenrirZeroZero** [comment]
I guess the elements are positioned with x/y coordinates? how aboaut adding an simple offset option for the top left and top right elements. That way we could get them to our position with some try and error :)
